data/scenario-02.json

```diff
@@ -0,0 +1,34 @@
+{
+  "locatorId": "c0000000002o",
+  "passport": {
+    "language": "en-gb",
+    "home": "http://www.bbc.co.uk/ontologies/passport/home/News",
+    "articleType": "news",
+    "genre": null
+  },
+  "publishedDateTime": "2018-06-04T00:00:00.000Z",
+  "lastUpdatedDateTime": "2018-06-05T12:00:00.000Z",
+  "seoHeadline": null,
+  "tags": {
+    "about": ["http://www.bbc.co.uk/things/3fbfd244-928b-498d-a585-9e39bd27fad3#id"],
+    "mentions": null
+  },
+  "metaDescription": null,
+  "blocks": [{
+    "blockId": "21cbb616-613e-4750-8c52-8a46c39456e0",
+    "type": "headline",
+    "content": "Lorem ipsum dolor sit amet, consectetur _adipiscing_ elit"
+  }, {
+    "blockId": "4b9ae65b-dc27-4cf1-82c6-36a24b005cbc",
+    "type": "image",
+    "aspectRatio": "1:1",
+    "relPath": "/cpsprodpb/439A/production/_100960371_syrians_and_asylum_v2-nc.png",
+    "copyrightHolder": "BBC",
+    "caption": "Praesent sagittis fringilla nibh ut vulputate. Ut _nec purus_ est. Vestibulum in eleifend nibh.",
+    "altText": "ICras dapibus ligula sit amet est aliquam, nec dignissim diam gravida."
+  }, {
+    "blockId": "da00008c-fc2b-44eb-8eae-30d65b26bd70",
+    "type": "paragraph",
+    "content": "Donec ut fermentum risus, eu sodales mauris. Quisque This is <sup>superscript</sup> vitae est sed nisl tincidunt facilisis. Nec vulputate metus sagittis./n/nQuisque quis libero bibendum, feugiat felis ut, dignissim tortor. Mauris metus lectus, lacinia sit amet massa ut, interdum convallis ipsum."
```

Comment:
Is `/n` meant to be a line-break?

Also, are we handling new paragraphs with a line-break? @benjaminhobbs

Reply:
We will ignore /n so we can strip it out or leave it as a test of out md parsing.